lib/types/youtube.js

```diff
@@ -23,5 +23,7 @@ export const parse = ([elm]) => {
   };
 };
 
-export const render = ({youtubeId}) =>
-  <iframe src={`https://www.youtube.com/embed/${youtubeId}`} />;
+export const render = ({youtubeId, width = 640, height = 480}) => renderIframe({
```

Comment:
but yeah seem that 16:9 is more common than 4:3, I'll update the default width/height to `640 x 360`